### ISSUE TITLE
Load model error using bloom-3b onnx model converted by optimum-cli

### ISSUE THREAD

**Issue by lywchaos** [huggingface/optimum#788]
### System Info

```shell
optimum: 1.6.4
python: 3.8.0
```

### Who can help?

@lewtun @JingyaHU

### Information

- [ ] The official example scripts
- [X] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [X] My own task or dataset (give details below)

### Reproduction

Using optimum-cli to convert bloom-3b model like this: `optimum-cli export onnx --model "bigscience/bloom-3b" /data/bloom3bonnx/ `  

Got this:  
![image](https://user-images.githubusercontent.com/109707080/219277080-36136e42-4add-4ac9-8390-b42e4211ea6b.png)

Then run this script to test:  
```python
from transformers import AutoTokenizer
from optimum.onnxruntime import ORTModelForCausalLM

tokenizer = AutoTokenizer.from_pretrained("/data/bloom3bonnx")
model = ORTModelForCausalLM.from_pretrained("/data/bloom3bonnx")
```

Got errors like:
```Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/liangyuanwei/miniconda3/envs/gatling/lib/python3.8/site-packages/optimum/onnxruntime/modeling_ort.py", line 581, in from_pretrained
    return super().from_pretrained(
  File "/home/liangyuanwei/miniconda3/envs/gatling/lib/python3.8/site-packages/optimum/modeling_base.py", line 341, in from_pretrained
    return from_pretrained_method(
  File "/home/liangyuanwei/miniconda3/envs/gatling/lib/python3.8/site-packages/optimum/onnxruntime/modeling_decoder.py", line 509, in _from_pretrained
    decoder_path = ORTModelDecoder.infer_onnx_filename(
  File "/home/liangyuanwei/miniconda3/envs/gatling/lib/python3.8/site-packages/optimum/onnxruntime/modeling_ort.py", line 377, in infer_onnx_filename
    raise FileNotFoundError(f"Could not find any ONNX model file for the regex {pattern} in {path}.")
FileNotFoundError: Could not find any ONNX model file for the regex (.*)?decoder((?!with_past).)*?\.onnx in /data/bloom3bonnx/.


```

### Expected behavior

Expect to successfully load and inference using optimum.onnxruntime

**Comment by fxmarty** [comment]
Thank you, please do the export as `optimum-cli export onnx --model "bigscience/bloom-3b" --for-ort --task causal-lm-with-past /data/bloom3bonnx/`.

Alternatively, you can pass `use_cache=False` in the `from_pretrained` to disable the reuse of past key values.

**Comment by xinbingzhe** [comment]
> Thank you, please do the export as `optimum-cli export onnx --model "bigscience/bloom-3b" --for-ort --task causal-lm-with-past /data/bloom3bonnx/`.
> 
> Alternatively, you can pass `use_cache=False` in the `from_pretrained` to disable the reuse of past key values.

I use the same  optimum  version .  I run like this
`bash
   optimum-cli export onnx --model bloomz-1b1 --for-ort --task causal-lm-with-past --framework pt bloom_onnx_v17
`
But I got this error
![image](https://user-images.githubusercontent.com/12292924/225845917-88f83337-c0d5-4fe7-ab77-81d4e4c45152.png)

**Comment by fxmarty** [comment]
Hi @xinbingzhe , which optimum version are you using?

Exporting with optimum==1.7.1 with `optimum-cli export onnx --model bigscience/bloomz-1b1 --no-post-process bloomz1b1_onnx` and torch==2.0, the export works fine to me. Should also work fine with torch==1.13.1.

**Comment by xinbingzhe** [comment]
> 

Thanks you,  Its helpful.  I solved after using torch==1.12.1.
